README.md

```diff
@@ -57,6 +57,8 @@ This module creates a script to configure and start the Teleport service on a se
 
 ### Output
  * [`teleport_bootstrap_script`]: String: The rendered script to add to the Instance cloud-init user data.
+ * [`teleport_config_cloudinit`]: String: The rendered Teleport config that you can add to the instance cloud-init user data
+ * [`teleport_service_cloudinit`]: String: The rendered Teleport systemd service that you can add to the instance cloud-init user data
```

Comment:
to be sure, you need to put these in a `write_files:` block right?

Reply:
yes